core/dropdowndiv.js

```diff
@@ -584,17 +605,9 @@ Blockly.DropDownDiv.hideWithoutAnimation = function() {
     clearTimeout(Blockly.DropDownDiv.animateOutTimer_);
   }
 
-  // Reset style properties in case this gets called directly
-  // instead of hide() - see discussion on #2551.
-  var div = Blockly.DropDownDiv.DIV_;
-  div.style.transform = '';
-  div.style.left = '';
-  div.style.top = '';
-  div.style.opacity = 0;
-  div.style.display = 'none';
-  div.style.backgroundColor = Blockly.DropDownDiv.DEFAULT_DROPDOWN_COLOR;
-  div.style.borderColor = Blockly.DropDownDiv.DEFAULT_DROPDOWN_BORDER_COLOR;
-
+  Blockly.DropDownDiv.clearContent();
+  Blockly.DropDownDiv.resetCss();
+  Blockly.DropDownDiv.owner_ = null;
```

Comment:
Note: I recently moved this so that's it after the onHide to give the onHide method the ability to interface with the dom element (eg: React to unmount). 
This will probably be resolved by a re-base but leaving a note to verify.